src/frontend/src/pages/Dashboard/Dashboard.jsx

```diff
@@ -8,7 +8,7 @@ function Dashboard() {
     const navigate = useNavigate();
 
     return (
-        <Grid fullWidth>
```

Comment:
this is from the docs and was recommended to use, but it's bugged with and fills the console with extra noise